### ISSUE TITLE
508-defect-2 [FOCUS MANAGEMENT]: Direct Deposit Component - focus must be managed during transition from read to edit in place and back again

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#18435]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## User Story or Problem Statement

* As a screen reader user, I want to hear a heading read aloud when I click the Update account information or Save buttons to switch into our out of edit mode.
* As a keyboard user, I want focus to be managed so I can quickly update my direct deposit information without having to press TAB or SHIFT + TAB a number of times to focus the correct element.

## Details

There are two key opportunities to manage focus on the Direct Deposit component so assistive tech and keyboard users have the best possible experience. I have attached two screenshots that identify where focus should go after clicking each button.

## Acceptance Criteria

- [ ] Focus is properly set on the heading after a button press or click
- [ ] Heading is properly announced to screen readers
- [ ] Heading does not have a visible focus halo

## WCAG or Vendor Guidance (optional)

* [Meaningful Sequence: Understanding SC 1.3.2](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-sequence.html)

## Screenshots or Trace Logs (optional)
![Screen Shot 2021-01-12 at 3 46 00 PM](https://user-images.githubusercontent.com/934879/104378701-06ebd300-54ee-11eb-8bd1-a62ac46cdb5d.png)

---
![Screen Shot 2021-01-12 at 3 46 10 PM](https://user-images.githubusercontent.com/934879/104378715-0d7a4a80-54ee-11eb-881c-20c5287f6dc1.png)
